daemon/src.cmake

```diff
@@ -301,7 +301,7 @@ set(QCOMMONLIST
     ${ENGINE_DIR}/sys/con_common.cpp
 )
 
-if (NOT APPLE)
+if (NOT APPLE AND (USE_CURSES OR WIN32))
```

Comment:
Yeah it is a bit much. 
It seems like windows must use curses enabled (effectively ignores USE_CURSES), Linux can use curses and apple must not use curses. So it seems to me that apple and windows shouldn't even have the option while linux is the only one that should be able to enable/disable curses.